### PR TITLE
Resolve when receiving None value

### PR DESCRIPTION
**Pull request by bsieber-mozilla**
This PR is looking to address issue #354 and uses the proposed solution by @jwhitlock

### CODE REVIEW

**Comment by bsieber-mozilla** [comment]
Next steps:  

- [ ] Bump version ( #357 )
- [ ] Deploy new version

File: ctms/app.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 """The CTMS application, including middleware and routes."""
 # pylint:disable = too-many-lines
+# pylint:disable = too-many-statements
```
Comment (jwhitlock) [inline ctms/app.py:3]: ouch! :hurtrealbad:

It may be worth breaking this into more files, but definitely outside of this PR.

**Review by jwhitlock — APPROVED** [review]
Thank you @bsieber-mozilla !

File: ctms/app.py (hunk 1 above)
Comment (bsieber-mozilla) [reply ctms/app.py:3]: https://github.com/mozilla-it/ctms-api/issues/360

Constructed that issue to keep track of this, thanks for the call out.

**Pull request merged**